### PR TITLE
Rickroll no ads- Harmless

### PR DESCRIPTION
**Pull request by J0K3RS-L4UGH**
A rickroll no ads

### CODE REVIEW

**Review by Concara3443 — CHANGES_REQUESTED** [review]
Can be shorter

File: payloads/library/prank/AllOperatingSystemRickroll/all [hunk 1]
```diff
@@ -0,0 +1,12 @@
+REM made by J0K3RS-L4UGH
+REM Basically it enters chrome and rickrolls with no ads
+GUI
+DELAY 2000
+STRING Chrome
+DELAY 2000
+ENTER
+DELAY 2000
+ALT d
+DELAY 2000
```
Comment (Concara3443) [inline payloads/library/prank/AllOperatingSystemRickroll/all:10]:
```suggestion
GUI r
```

**Review by Concara3443 — CHANGES_REQUESTED** [review]

**Comment by J0K3RS-L4UGH** [comment]
> Can be shorter

How?

**Comment by J0K3RS-L4UGH** [comment]
This is made to work in all operating systems, including ChromeOS and Linux

**Pull request merged**